sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Microsoft.Azure.WebPubSub.AspNetCore.csproj

```diff
@@ -12,9 +12,13 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="System.Memory.Data" />
```

Comment:
This file seems to set ``$(RequiredTargetFrameworks)`` to ``netcoreapp3.1`` should it add ``net6.0``?